content/guide/netlify.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Netlify for Quire
+weight: 215
+type: essay
+---
+
+## Netlify 
+
+{{< q-figure id="netlify-logo" >}}
+
+Netlify is a web developer platform that multiplies productivity.
+
+By unifying the elements of the modern decoupled web, from local development to advanced edge logic, Netlify enables a 10x faster path to much more performant, secure, and scalable websites and apps.
+
+Our bet on the JAMstack is quickly coming true. The web is rapidly changing away from monolithic to decoupled apps, and web developers are storming ahead with more power than ever. Netlify is built to cater to that movement, and in just a few years we’ve on-boarded more than half a million businesses and developers, and are building and serving millions of web projects daily around the globe.
+
+## Steps to deploy to Netlify
+
+- As stated in the previous page add your code to Github repositiory
+- Next sign up to Netlify here -> https://app.netlify.com/signup
+- Connect your Netlify account to your Github or Gitlab account
+
+
+### Netlify Build Configuration
+
+Once you accounts are connected you will asked to provide a `Production` directory or `Build` directory. Instead of doing this, which is fine, my recommendation is heading back to your repository and creating a `netlify.toml` file which will run commands from the root directory. These commands are set in the scripts object in the your `package.json` file. Currently the `package.json` file is not in the root directory so make a copy of the one in your theme folder. In quire-starter-theme the path is `themes/quire-starter-theme/package.json`. 
```

Comment:
I never realized the `package.json` file you're adding is just a duplicated and slightly altered version of the one in the theme. I don't suppose there's any chance we could just alter the theme one from the get go and then have Netlify run something like `npm run themes/quire-starter-theme/build`? So that users didn't have to duplicate?

If not, my next question would be, could we go ahead and add the build script(s) to the `package.json` file in the theme now, so that users would only need to make a copy of the file and not also add the script in manually?

Reply:
Let me see yeah I was sure if that would work because the referenced `node_modules` would be in the wrong place, so I thought the reference would be gone, so I never actually tried it. I will take a look today and let you know. Yeah it would be much better to have one file instead of multiple.

Reply:
so this 
```bash 
npm --prefix themes/quire-starter-theme run build
```
works ! So that is great !

Just need to remove the `cd themes/quire-starter-theme/` from the script. 

Seems to work fine.

```bash 
webpack --config webpack/webpack.config.prod.js && cd ../../ && hugo --config config.yml,config/site.yml
```

Reply:
We can use this config going forward.